api/internal/core/store/store.go

```diff
@@ -136,8 +136,9 @@ func (s *GenericStore) Init() error {
 	return nil
 }
 
-func (s *GenericStore) Get(key string) (interface{}, error) {
+func (s *GenericStore) Get(ctx context.Context, key string) (interface{}, error) {
 	ret, ok := s.cache.Load(key)
+	_ = ctx
```

Comment:
If `ctx` is not in use, just use dummy variable when define the function argument.

Reply:
fixed.